CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the IBM® Db2® Plug-in for Zowe CLI will be documented in this file.
 
+## `4.0.8` 
```

Comment:
Header should be "Recent Changes" instead of "`4.0.8`". When the PR is merged the version automatically gets updated here.